chronograf/bolt/dashboards.go

```diff
@@ -13,7 +13,7 @@ import (
 var _ chronograf.DashboardsStore = &DashboardsStore{}
 
 // DashboardsBucket is the bolt bucket dashboards are stored in
-var DashboardsBucket = []byte("Dashoard")
+var DashboardsBucket = []byte("Dashboard")
```

Comment:
not sure if this is dead code or not, but worried it might have larger impacts.

Reply:
@glinton I feel like you might be able to answer if this has larger impacts

Reply:
last i knew it was still being used and changing it would cause old dashboards to go missing. if it's worth it to update this spelling, a migration would need to be written

Reply:
The only place i found this to be used was in `cmd/chronograf-migrator`, I think this particular change should be reverted since chronograf still stores dashboards in that misspelled bucket.
 - [migrator code](https://github.com/influxdata/influxdb/blob/63974e8fe70f2be29704116af1e7c9aac80c67ee/cmd/chronograf-migrator/main.go#L33)
 - [chronograf code](https://github.com/influxdata/chronograf/blob/1f1e1eebc965fbafb41772b5c6373d99ad7e4b36/kv/kv.go#L17)

If `cmd/chronograf-migrator` is not the preferred method to migrate from v1, then all this code could be removed.